src/business.ts

```diff
@@ -281,16 +281,25 @@ export class BusinessApi {
   }
 
   /*
-   * Function to take a Business token Id, send that to Marqeta, and have a
-   * list of Business transition statuses returned.
+   * Function to take a list of search parameters, which include a Business
+   * token Id, send that to Marqeta, and have a list of Business transition
+   * statuses returned based on the optional parameters.
    */
-  async listTransition(token: string): Promise<{
+  async listTransition(search: {
+    token?: string,
+    count?: number,
+    startIndex?: number,
+    searchType?: string,
+    fields?: string[],
+    sortBy?: string,
+  } = {}): Promise<{
     success: boolean,
     body?: TransitionList,
     error?: MarqetaError,
   }> {
     const resp = await this.client.fire('GET',
-      `businesstransitions/business/${token}`,
+      `businesstransitions/business/${search.token}`,
+      search,
```

Comment:
Given this, it's quite possible for `search.token`... or in this context, `token`, to be `undefined`... you _probably_ want to check that before making the call...

Reply:
You still didn't construct the body... why not use those unused vars:

```typescript
     const resp = await this.client.fire('GET',
       `businesstransitions/business/${token}`,
       { token, count, startIndex, searchType, fields, sortBy },
     }
```